### PR TITLE
data: Pyth0nHere

### PR DESCRIPTION
**Pull request by Pyth0nHere**

### CODE REVIEW

File: data/Pyth0nHere.json
Comment (krishstwt) [inline data/Pyth0nHere.json:24]: Trailing comma not allowed in JSON

```suggestion
    }
```

**Review by krishstwt — CHANGES_REQUESTED** [review]

**Review by krishstwt — APPROVED** [review]

**Review by Pyth0nHere — COMMENTED** [review]
Merge it

File: data/Pyth0nHere.json
Comment (RohanSasne) [inline data/Pyth0nHere.json:18]:
```suggestion
      "icon": "FaLinkedin"
```

Icons are case sensitive :)

**Review by RohanSasne — CHANGES_REQUESTED** [review]

**Review by Pyth0nHere — COMMENTED** [review]
made changes.

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Comment by SaraJaoude** [comment]
> Merge it

@Pyth0nHere please consider that this sounds unnecessarily aggressive. Also this cannot be merged yet as the JSON is not valid - please look at the GitHub Actions in future.

**Comment by SaraJaoude** [comment]
@CBID2 @krishstwt as I have mentioned to you both in the past please do not approve a PR before reviewing it properly.  In this case the GitHub Actions were not passing as the JSON is still invalid.

File: data/Pyth0nHere.json
Comment (SaraJaoude) [inline data/Pyth0nHere.json:25]: add the missing bracket:
```suggestion
  ]
  }
```

**Comment by CBID2** [comment]
> @CBID2 @krishstwt as I have mentioned to you both in the past please do not approve a PR before reviewing it properly.  In this case the GitHub Actions were not passing as the JSON is still invalid.

Oh no I didn't catch that other brackets. 🤦🏾‍♀️

**Review by SaraJaoude — APPROVED** [review]
I have added the missing comma for a valid JSON and committed this - as the GitHub Action is now passing I will merge this PR.

**Comment by Pyth0nHere** [comment]
> > Merge it
> 
> @Pyth0nHere please consider that this sounds unnecessarily aggressive. Also this cannot be merged yet as the JSON is not valid - please look at the GitHub Actions in future.

@SaraJaoude  Thank you for your feedback. I apologize if my previous message came across as unnecessarily aggressive. I'll make sure to be more mindful of my tone in future communications. Regarding the merge, I understand that the JSON is currently invalid, and I'll take a closer look at the GitHub Actions to resolve any issues before proceeding with the merge. Your input is appreciated, and I'll ensure better communication going forward. Let me know if there's anything specific you'd like me to address or if you have any further concerns.

**Pull request merged**
